quickevent/app/quickevent/plugins/CardReader/src/cardreaderplugin.cpp

```diff
@@ -449,7 +449,7 @@ void CardReaderPlugin::setStartTimeForNextLeg(int relay_id, int prev_leg, int pr
 	q.exec("UPDATE runs SET startTimeMs=" + QString::number(prev_finish_time)
 		   + " WHERE relayId=" + QString::number(relay_id)
 		   + " AND leg=" + QString::number(prev_leg+1)
-		   + " AND COALESCE(startTimeMs, 0)=0"
+		   //+ " AND COALESCE(startTimeMs, 0)=0" faster runner can readout after the slower one, hence this check cannot be used
```

Comment:
How can this situation happen? The two runners on the same leg start together, thus the faster one always finishes first.

Reply:
they can finnish at around the same time and the slower one can reach the readout station first

Reply:
_reverted_ toto přepisovalo startovní časy závodníků startujících z hambastartu.